ProjectManagmentSystem/src/Modules/ProjectsModule/components/ProjectsData/ProjectsData.tsx

```diff
@@ -49,50 +49,58 @@ export default function ProjectsData() {
   
   return (
     <>
-      <div className="compTitle  my-5 bg-white p-4 shadow-lg">
+      <div className="add-headers rounded-3 my-5 bg-white p-4 shadow-lg">
         <span>
-         <button onClick={navigatetoProjects} className="btn"> <i  className="fa fa-chevron-left"></i>
-          View all Projects</button>
+          <i onClick={navigatetoProjects} className="fa fa-chevron-left me-2"></i>
```

Comment:
This should be a Link, since the main purpose of it is navigation.

Reply:
done